### PR TITLE
[ja] Fix: link to ja/docs/concepts/extend-kubernetes/operator/ in ja/docs/concepts/extend-kubernetes/

### PR DESCRIPTION
**Pull request by nuka137**
Fix the link to `ja/docs/concepts/extend-kubernetes/operator/`

### CODE REVIEW

**Comment by riita10069** [comment]
Thank you for your PR. It seems to only contain trivial edits. Please read our trivial edits policy.
https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits

There are about four other links can be modified in the same file that.
It would be very helpful if you could fix that as well.

**Comment by nuka137** [comment]
@riita10069 

Thanks for your advice. I fixed other missing links.
Could you check them?

File: content/ja/docs/concepts/extend-kubernetes/_index.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ Kubdernetesはいくつかのビルトイン認証方式をサポートしてい
 
 [認証](/ja/docs/reference/access-authn-authz/authentication/)は、全てのリクエストのヘッダーまたは証明書情報を、リクエストを投げたクライアントのユーザー名にマッピングします。
 
-Kubernetesはいくつかのビルトイン認証方式と、それらが要件に合わない場合、[認証Webhook](/docs/reference/access-authn-authz/authentication/#webhook-token-authentication)を提供します。
+Kubernetesはいくつかのビルトイン認証方式と、それらが要件に合わない場合、[認証Webhook](ja/docs/reference/access-authn-authz/authentication/#webhook-token-authentication)を提供します。
```
Comment (riita10069) [inline content/ja/docs/concepts/extend-kubernetes/_index.md:118]: Could you start `/`?

Comment (nuka137) [reply content/ja/docs/concepts/extend-kubernetes/_index.md]: Fixed it. Thanks

**Comment by nasa9084** [comment]
Very sorry to late review...
/approve

**Pull request merged**